app/views/spree/shared/_taxonomies.html.erb

```diff
@@ -1,16 +1,40 @@
-<% max_level = Spree::Config[:max_level_in_taxons_menu] || 1 %>
+<%# max_level = Spree::Config[:max_level_in_taxons_menu] || 1 %>
 
-<nav id="taxonomies" class="sidebar-item" data-hook aria-label="taxonomies">
-  <% @taxonomies.each do |taxonomy| %>
-    <% cache [I18n.locale, taxonomy, max_level, @taxon] do %>
-      <% if taxonomy.root.children.any? %>
+<!-- <nav id="taxonomies" class="sidebar-item" data-hook aria-label="taxonomies">
+  <%# @taxonomies.each do |taxonomy| %>
+    <%# cache [I18n.locale, taxonomy, max_level, @taxon] do %>
+      <%# if taxonomy.root.children.any? %>
         <div class="mt-4">
           <h4 class='taxonomy-root h5'>
-            <%= Spree.t(:shop_by_taxonomy, taxonomy: taxonomy.name) %>
+            <%#= Spree.t(:shop_by_taxonomy, taxonomy: taxonomy.name) %>
           </h4>
-          <%= taxons_tree(taxonomy.root, @taxon, max_level) %>
+          <%#= taxons_tree(taxonomy.root, @taxon, max_level) %>
         </div>
+      <%# end %>
+    <%# end %>
+  <%# end %>
+</nav> -->
+
+ <div id="taxonomies" data-hook aria-label="taxonomies">
+  <% max_level =  3 %>
+  <!-- <nav id="taxonomies"  data-hook> -->
+  <h4>Categories:</h4>
+  <% if @taxon.present? %>
```

Comment:
@Umer, the taxons should show as per users. As i cant see the taxons collections by user. I think we missed that case. This is actual requirement of this ticket.